packages/kolibri-api/index.js

```diff
@@ -1,6 +1,6 @@
 // Communication with Kolibri frontend:
 
-export async function getThumbnail(node) {
+export function getThumbnail(node) {
```

Comment:
Why this change? I don't get any errors because of this. And the lint in kolibri-api does nothing, I removed it in my maintenance PR https://github.com/endlessm/kolibri-explore-plugin/pull/74/files#diff-e9e2f9447b84f3e3dd465bcba73a598a60480fd5a788853108b5c5e412b05057L6-L11

Reply:
I get a lint error because of this rule: https://eslint.org/docs/rules/require-await

Reply:
Right now there's no lint in the `kolibri-api` package, so that's the case there's no error, but if you run the command `yarn eslint packages/kolibri-api/index.js` you will see this output:

```
/home/danigm/Projects/work/endless/kolibri/kolibri-explore-plugin/packages/kolibri-api/index.js
  3:8  error  Async function 'getThumbnail' has no 'await' expression  require-await

✖ 1 problem (1 error, 0 warnings)
```

We can add a lint command to this package to catch these issues.

Reply:
Good point, I have opened a ticket for tracking this: https://phabricator.endlessm.com/T32022